.github/workflows/on_pr_opened.yaml

```diff
@@ -34,6 +34,9 @@ jobs:
           image_name: ${{ env.APP_NAME }}
           github_username: ${{ github.repository_owner }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
+          app_title: Common Hosted Forms - PR-${{ github.event.number }}
+          route_path: /pr-${{ github.event.number }}
+          app_contact: ${{ secrets.VUE_APP_CONTACT }}
```

Comment:
Would you add `app_howtourl: ${{ secrets.VUE_APP_HOWTOURL }}` and `app_chefstoururl: ${{ secrets.VUE_APP_CHEFSTOURURL }}` here?

Reply:
I don't think we should this in 

> 



> Would you add `app_howtourl: ${{ secrets.VUE_APP_HOWTOURL }}` and `app_chefstoururl: ${{ secrets.VUE_APP_CHEFSTOURURL }}` here?

I don't this is the right file or place to do this